cmd/root.go

```diff
@@ -73,7 +73,7 @@ func init() {
 
 	flags := rootCmd.PersistentFlags()
 
-	flags.StringVar(&rootOptions.CfgFile, "config", "", "config file (default is $HOME/.banzai/config.yaml)")
+	flags.StringVar(&rootOptions.CfgFile, "config", "", "config file (default is $BANZAICONFIG or $HOME/.banzai/config.yaml)")
```

Comment:
```suggestion
	flags.StringVar(&rootOptions.CfgFile, "config", "", "config file (default is $HOME/.banzai/config.yaml)")
```

`$BANZAICONFIG` is not a "default". It should be a documented feature instead.

Reply:
Actually, it is a default. The config file named in $BANZAICONFIG, or if it's unset, the one in .banzai will be used if you don't specify this option. I would anyhow mention this near the `--config` option's short description. I would leave it like now if you don't have a better idea that mentions the env var too.

Reply:
Default is something that you don't specify. You specify env vars, therefore it's not a default for me.

Since `--config` overrides env vars anyway, mentioning the env var here does not really add any value IMO.

Kubectl does not mention it either:

```
--kubeconfig='': Path to the kubeconfig file to use for CLI requests.
```

If you really want to add it somewhere, add it to the long definition of the command, but it doesn't belong here IMHO.

Reply:
i think this helpful here, even i had to check the code repeatedly in the past to find out the name of this var. this is the first place where i would seek this as a user

Reply:
I'm arguing that it is a default or that it should be documented together with a flag, not that it should be documented.